### DISCUSSION TITLE
easy fix

### DISCUSSION THREAD

**Discussion by rwg100** [3kh0/ext-remover#389] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
[https://docs.google.com/document/d/1PyjkcA4CVbXYX4gYIh1gM6wPz1SDFr179ReIonT5Q5k/edit#](https://github.com/3kh0/ext-remover/discussions/url)

so basicly it confuses all of your extensions so they dont work
and for the last step you are NOT supposed to add your own account you are supposed to add your SCHOOL acocunt back. not your personal. and you have to repaeat every time you log out or restart.

**Comment by Sxri** [comment]
Dumb method, also very buggy on certain versions

**Reply by Aka-but-nice** [reply]
True, @Sxri do you want to help with the ingot for 106 bypass?

**Reply by Aka-but-nice** [reply]
I'm using updated point-blank

**Reply by Sxri** [reply]
> True, @Sxri do you want to help with the ingot for 106 bypass?

Sure, However, I'm busy frequently so I'm usually only available at school.

**Reply by Aka-but-nice** [reply]
Alr, it's pretty easy anyway.
